client/packages/system/src/Patient/api/hooks/document/usePatient.ts

```diff
@@ -1,7 +1,7 @@
 import { useQuery } from '@openmsupply-client/common';
 import { usePatientApi } from '../utils/usePatientApi';
 
-export const usePatient = (nameId: string) => {
+export const usePatient = (nameId: string | undefined) => {
```

Comment:
```suggestion
export const usePatient = (nameId?: string) => {
```

Reply:
This was intentionally. It doesn't make sense to call usePatient without parameter but the user might provide a string that is undefined, e.g. because nameId is not know yet. This syntacx enforces that a parameter must be provided.

Reply:
ah, right.